man/cc-test-reporter-format-coverage.1.md

```diff
@@ -59,7 +59,7 @@ As generated by **coverage.py**.
 
 As generated by **phpunit --coverage-clover**.
 
-## ./cover.out *Go*
+## go test -coverprofile=c.out *Go*
```

Comment:
@davehenton 
what do you think about leaving this as `c.out *Go*`

and changing line 64 to As generated by `go test -coverprofile=c.out`